teams/vsp/teams/operations/daytime-oncall-support-role.md

```diff
@@ -3,40 +3,42 @@
 The Operations team can be “interrupted” by unscheduled requests that come in from the VFS or VSP teams, leadership, monitoring, and other teams and stakeholders at the VA. We want to find some ways to reduce the interruptions for individual team members and clarify who is responsible for handling each request by funneling individual tasks through one person. This person will handle things that can be handled directly and efficiently and delegate things that cannot. Some specific examples of the kind of requests handled by this developer are:
 
 * Normal oncall [responsibilities](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/tree/master/OnCall) 
-* Monitoring oncall channel
+* Monitoring #oncall channel (and others listed below)
 * Requests for assistance from the VFS or VSP teams
-* Support requests in the support channels
+* Support requests in the support channel (#vfs-platform-support)
 * Responding to and reviewing any internal or external DevOps PRs
-* Respond and grant [approved access](https://github.com/department-of-veterans-affairs/devops/blob/master/docs/operations/Internal_Onboarding_Instructions.md) to requests for access in Zenhub or remove access per [offboarding requests](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/contractoroffboarding.md).
+* Respond and grant approved access (**is there an updated link to add here?**) to requests for access in Zenhub or remove access per [offboarding requests](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/contractoroffboarding.md)
```

Comment:
The original link 404s 
[approved access](https://github.com/department-of-veterans-affairs/devops/blob/master/docs/operations/Internal_Onboarding_Instructions.md)
Is there something else we want to put here?

Reply:
Perhaps https://vfs.atlassian.net/wiki/spaces/OT/pages/1792114735/Onboard+New+Team+Members+Granting+Access+to+Internal+Tools

Reply:
Offboarding link is out of date but the  Offboarding Request template is valid